### DISCUSSION TITLE
how many bits switches actually have?

### DISCUSSION THREAD

**Discussion by nablabla** [ExpressLRS/ExpressLRS#1509] [Q&A] [answered]
Hi, I am quite frustrated that I can not use ELRS for gliders, since you need full 10bit channels for both ailerons and also for the flaps, so an 8 channel mode (with round robbin) would be really necessary for planes.

Anyways my question is that here https://www.expresslrs.org/2.0/software/switch-config/ it says in hybrid mode we get:
AUX2-AUX8 | 6 or 7-bit / 64 or 128-position | Round-robin
So what now? is it 6 or 7 bits? Can I set this or what does it depend on? 7 bits might be borderline acceptable for flaps (but than they need mechanical fine tuning) and auxiliary motor

**Comment by pkendall64** [comment, marked as answer]
If you read further down that page https://www.expresslrs.org/2.0/software/switch-config/#aux-2-8-wide you will see that they are 6 or 7 bit depending on the telemetry ration you select.
The first 4 channels are still the full 10bit channels as well.
Also, the mode you are after is called Wide switch mode.

**Reply by nablabla** [reply]
Thank you. (although i need 8x 10bit)

**Reply by pkendall64** [reply]
That is something that _may_ come later with FLRC, but I don't think anyone is working on it currently.

**Comment by nablabla** [comment]
eventually elrs can do but anything! I am a sw dev so I can do it myself @,@
